documentation/writing-policies-generate.md

```diff
@@ -75,7 +84,7 @@ spec:
             purpose: mongo
 ````
 
-In this example new namespaces will receive 2 new resources after its creation:
+In this example each namespaces will receive 2 new resources if label match:
```

Comment:
Should be "each namespace" .

Please document the behavior on matched pre-existing resource(namespace).

Reply:
I think that is clear when we say each namespace